src/components/atoms/CyButton.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <button :class="`cy-button cy-button--${kind}`">
+    <div class="_button-text">
+      <img
+        src="cy-icon-black-transparent.svg"
+        width="16"
+        style="vertical-align: middle; margin-right: 8px; margin-bottom: 2px"
+      />{{ buttonLabel }}
+    </div>
+  </button>
+</template>
+
+<style lang="sass" scoped>
+// $
+._button-text
+  +t-h5()
```

Comment:
@fergusmeiklejohn FYI
This "sass mixin" here can be used because adding `// $` to the top of the `<style>` section in Quasar, makes sure that Quasar automatically makes any CSS from inside the `quasar.variables.sass` file usable in this component.

Since we now have our typography file in there you can use any of those typography mixins like so.